### DISCUSSION TITLE
Internal server error still an issue

### DISCUSSION THREAD

**Discussion by JamesArthurHolland** [community/community#13265] [Other Feature Feedback, Questions, & Ideas]
remote: Internal Server Error
To github.com:JamesArthurHolland/______.git
 ! [remote rejected] main -> main (Internal Server Error)
error: failed to push some refs to 'git@github.com:JamesArthurHolland/uphold-tech-test.git'

I tried this on multiple repos. Same issue

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
